test/test_collection.py

```diff
@@ -197,12 +197,6 @@ def test_create(self):
             lambda: "create_test_no_wc" in db.list_collection_names(),
             "create create_test_no_wc collection",
         )
-        db.create_test_no_wc.drop()
```

Comment:
We should keep this and catch the warning.

Reply:
Correct, when we deprecate something we keep the tests and also add an extra assertion to ensure the warning is actually raised.